TypeCobol.Analysis/Cfg/ControlFlowGraphBuilder.DeclarativesContext.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System.Collections.Generic;
+
+namespace TypeCobol.Analysis.Cfg
+{
+    public partial class ControlFlowGraphBuilder<D>
+    {
+        /// <summary>
+        /// Declarative context class.
+        /// </summary>
+        internal class DeclarativesContext
+        {
+            /// <summary>
+            /// Current Block before all declaratives sections and paragraphs.
+            /// </summary>
+            internal BasicBlockForNode CurrentBlock;
```

Comment:
Rename as `StartBlock` ?

Reply:
In CFG a Start Block is in fact a Root Block, this is not always the case here. But ok, it is like the SourceBlock. or OriginBlock.

Reply:
done